packages/react/src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -54,28 +80,30 @@ const BarChartCard = ({
   className,
   domainRange,
   timeRange,
+  showTimeInGMT,
+  tooltipDateFormatPattern,
   ...others
 }) => {
-  const { noDataLabel } = i18n;
+  // need to deep merge the nested content default props as default props only uses a shallow merge natively
+  const contentWithDefaults = useMemo(() => defaultsDeep(content, defaultProps.content), [content]);
```

Comment:
this is bad because it mutates content, need to do `defaultsDeep({},content,defaultProps.content)